File: VB/XtraReport1.vb
Imports System.Drawing
Imports System.Drawing.Printing
Imports DevExpress.XtraPrinting
Imports DevExpress.XtraReports.UI

Public Partial Class XtraReport1
    Inherits XtraReport

    Public Sub New()
        InitializeComponent()
        Dim myStyle As XRControlStyle = New XRControlStyle With {.Name = "MyStyle1", .BackColor = Color.LightBlue, .BorderColor = Color.LightGray, .Borders = BorderSide.Top, .BorderWidth = 2F, .Font = New Font("Segoe Script", 16), .ForeColor = Color.Green, .TextAlignment = TextAlignment.TopCenter}
        StyleSheet.Add(myStyle)
    End Sub

    Private Sub xrLabel1_BeforePrint(ByVal sender As Object, ByVal e As System.ComponentModel.CancelEventArgs)
        CType(sender, XRLabel).StyleName = "MyStyle1"
    End Sub

    Private Sub XrLabel2_BeforePrint(ByVal sender As Object, ByVal e As System.ComponentModel.CancelEventArgs)
        ApplyAppearanceSettings(CType(sender, XRLabel))
    End Sub

    Private Sub XrLabel3_BeforePrint(ByVal sender As Object, ByVal e As System.ComponentModel.CancelEventArgs)
        ApplyAppearanceSettings(CType(sender, XRLabel))
        ResetStyle(CType(sender, XRLabel))
    End Sub

    ' Assign custom appearance settings to a control.
    Private Sub ApplyAppearanceSettings(ByVal label As XRLabel)
        label.BackColor = Color.Orange
        label.BorderColor = Color.DarkGray
        label.Borders = BorderSide.All
        label.BorderWidth = 0.5F
        label.Font = New Font(label.Parent.Font, FontStyle.Bold)
        label.ForeColor = Color.White
        label.TextAlignment = TextAlignment.MiddleRight
    End Sub

    ' Reset appearance settings and use parent control settings.
    Private Sub ResetStyle(ByVal label As XRLabel)
        label.ResetBackColor()
        label.ResetBorderColor()
        label.ResetBorders()
        label.ResetBorderWidth()
        label.ResetFont()
        label.ResetForeColor()
        label.ResetPadding()
        label.ResetTextAlignment()
    End Sub
End Class

File: CS/XtraReport1.cs
using System;
using System.Drawing;
using System.Drawing.Printing;
using DevExpress.XtraPrinting;
using DevExpress.XtraReports.UI;

public partial class XtraReport1 : XtraReport {
    public XtraReport1() {
        InitializeComponent();

        XRControlStyle myStyle = new XRControlStyle {
            Name = "MyStyle1",
            BackColor = Color.LightBlue,
            BorderColor = Color.LightGray,
            Borders = BorderSide.Top,
            BorderWidth = 2f,
            Font = new Font("Segoe Script", 16),
            ForeColor = Color.Green,
            TextAlignment = TextAlignment.TopCenter,

    };
        this.StyleSheet.Add(myStyle);
    }


    private void xrLabel1_BeforePrint(object sender, System.ComponentModel.CancelEventArgs e) {
        ((XRLabel)sender).StyleName = "MyStyle1";

    }
    private void XrLabel2_BeforePrint(object sender, System.ComponentModel.CancelEventArgs e) {
        ApplyAppearanceSettings((XRLabel)sender);
    }

    private void XrLabel3_BeforePrint(object sender, System.ComponentModel.CancelEventArgs e) {
        ApplyAppearanceSettings((XRLabel)sender);
        ResetStyle((XRLabel)sender);
    }

    // Assign custom appearance settings to a control.
    private void ApplyAppearanceSettings(XRLabel label) {
        label.BackColor = Color.Orange;
        label.BorderColor = Color.DarkGray;
        label.Borders = BorderSide.All;
        label.BorderWidth = 0.5f;
        label.Font = new Font(label.Parent.Font, FontStyle.Bold);
        label.ForeColor = Color.White;
        label.TextAlignment = TextAlignment.MiddleRight;
    }

    // Reset appearance settings and use parent control's settings.
    private void ResetStyle(XRLabel label) {
        label.ResetBackColor();
        label.ResetBorderColor();
        label.ResetBorders();
        label.ResetBorderWidth();
        label.ResetFont();
        label.ResetForeColor();
        label.ResetPadding();
        label.ResetTextAlignment();
    }
}
